src/pages/my_page/user_profile/MyProfile.tsx

```diff
@@ -296,20 +297,41 @@ export const MyProfile = () => {
                     </Field>
                   </Grid>
                 </Grid>
-
-                <Box
-                  sx={{
-                    gridArea: 'profile-picture',
-                    display: 'flex',
-                    flexDirection: 'column',
-                    alignItems: 'center',
-                    mb: { xs: '1rem', lg: 0 },
-                  }}>
-                  <Typography variant="h3" sx={{ mb: '1rem' }}>
-                    {t('my_page.my_profile.profile_picture')}
-                  </Typography>
-                  <ProfilePictureUploader personId={personId} />
-                </Box>
+                <FormBox sx={{ mb: '1rem' }}>
+                  <Box
```

Comment:
`FormBox` er en vertical flex, men her har den bare en `Box` (div) som igjen får egen styling, som definerer en ny vertical flex som sine children bruker. Virker som den første da kan fjernes, og ev. kan `mb` settes i den andre i stedet?
```suggestion
                <Box
```